<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>515</width>
      <height>223</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Строка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="value_row"/>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Столбец</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="value_col"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>                                    Значение ячейки</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Строковое значение</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cell_val"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Число</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cell_val_int"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Формула начала</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Строка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="value_row_func_st">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Столбец</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="value_col_func_st"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Формула конца</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Строка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="value_row_func_end"/>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Столбец</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="value_col_func_end"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>309</width>
      <height>123</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QPushButton" name="create_table">
       <property name="text">
        <string>Создать</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Количество строк</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Создать таблицу</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="coll_n"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Количество столбцов</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать значение ячейки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Сумма</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Произведение</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="value_col_2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Среднее значение</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="value_col_4"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="value_col_3"/>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>         Ячейка с формулой результат</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="save">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="save_2"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="save"/>
  </widget>
  <action name="save_2">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
